doc/cjdns.md

```diff
@@ -0,0 +1,95 @@
+# CJDNS support in Dash Core
+
+It is possible to run Dash Core over CJDNS, an encrypted IPv6 network that
```

Comment:
have you actually try it/have you actually test it?

Reply:
Would require me to spin up CJDNS nodes and test it.

Unit and functional tests seem fine (tested on IPv6-capable Docker container with `test_ipv6_local` patch reverted) and CJDNS doesn't require much handling beyond normal IPv6 except simply being aware of it, that was one of the reasons the PR was adopted upstream (see [comment](https://github.com/bitcoin/bitcoin/pull/23077#issuecomment-926626650)), BIP155 had already introduced rudimentary logic acknowledging CJDNS.

Reply:
Have confirmed that CJDNS works and that they're flagged correctly